src/main/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/RoleAssignmentFilteringResult.java

```diff
@@ -1,31 +1,45 @@
 package uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol;
 
-import org.apache.commons.lang3.tuple.Pair;
-
 import java.util.List;
 import java.util.stream.Collectors;
+import org.apache.commons.lang3.tuple.Pair;
 
 public class RoleAssignmentFilteringResult {
 
-    private final List<Pair<RoleAssignment, RoleMatchingResult>> roleAssignmentRoleMatchingResults;
+    private final List<Pair<RoleAssignment, RoleMatchingResult>> roleMatchingResults;
```

Comment:
Is `roleMatchingResults` the proper variable name when it's a list of pairs which include `RoleMatchingResult`, not a list of `RoleMatchingResult`?

Reply:
renamed to roleAssignmentMatchingResults